lib/MusicBrainz/Server/Report/PlaceReport.pm

```diff
@@ -3,6 +3,13 @@ use Moose::Role;
 
 with 'MusicBrainz::Server::Report::QueryReport';
 
+sub _load_extra_place_info {
+    my ($self, @places) = @_;
+
+    $self->c->model('Area')->load(@places);
+    $self->c->model('Area')->load_containment(map { $_->area } @places);
+}
+
```

Comment:
Is this for clarity, or is it a direct requirement of the JSON changes? I like it, anyway, it makes it a lot more clear, I just wasn't sure if it's required.

Reply:
It's a requirement because IIRC we had some reports overriding `inflate_rows` from the parent and loading more info on the output -- which is now impossible because `inflate_rows` returns plain hash refs, not blessed entities.